openwisp-openwrt-monitoring/Makefile

```diff
@@ -0,0 +1,77 @@
+# openwisp.org
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openwisp-monitoring
+PKG_VERSION:=$(shell cat ../VERSION)
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)/$(PKG_SOURCE_SUBDIR)
+PKG_LICENSE:=GPL-3.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/openwisp-openwrt-monitoring/default
+	TITLE:=OpenWISP Monitoring
+	CATEGORY:=Network
+	SECTION:=base
+	SUBMENU:=openwisp
+	PKGARCH:=all
+	MAINTAINER:=Federico Capoano <f.capoano@openwisp.io>
+	URL:=http://openwisp.org
+endef
+
+Package/openwisp-openwrt-monitoring=$(call Package/openwisp-openwrt-monitoring/default)
+
+define Package/openwisp-netjson-monitoring/default
+	TITLE:=NetJson Monitoring
+	CATEGORY:=Network
+	SECTION:=base
+	SUBMENU:=openwisp
+	DEPENDS:=+libubus-lua +lua-cjson
+	PKGARCH:=all
+	MAINTAINER:=Federico Capoano <f.capoano@openwisp.io>
+	URL:=http://openwisp.org
+endef
+
+define Build/Compile
+endef
+
+Package/openwisp-netjson-monitoring=$(call Package/openwisp-netjson-monitoring/default)
+
+define Package/openwisp-openwrt-monitoring/conffiles
```

Comment:
let's keep it simple: replace `openwisp-openwrt-monitoring` with `openwisp-monitoring`, `openwisp-netjson-monitoring` with `netjson-monitoring`.